app/services/search/base.rb

```diff
@@ -41,8 +41,21 @@ def document_count
         Search::Client.count(index: self::INDEX_ALIAS).dig("count")
       end
 
+      def search_documents(params:)
```

Comment:
could we add a default of `{}` here? or you don't want to risk that by chance someone is able to search all documents?

Reply:
I think it's better to require params to prevent that from accidentally happening.